pkg/sdn/plugin/node.go

```diff
@@ -272,11 +272,13 @@ func (node *OsdnNode) Start() error {
 				continue
 			}
 			if vnid, err := node.policy.GetVNID(p.Namespace); err == nil {
-				node.policy.RefVNID(vnid)
+				node.policy.EnsureVNIDRules(vnid)
 			}
 		}
 	}
 
+	go kwait.Forever(node.syncVNIDRules, 24*time.Hour)
```

Comment:
So this will sync once a day; seems like on a large cluster we could build up quite a list of stale VNID rules.  Should we do it like once an hour or 3 or something instead?